### ISSUE TITLE
getpubmedtbl can't generate the date

### ISSUE THREAD

**Issue by rusher321** [yufree/scifetch#1]
Hey, Miao ：

Thanks for your helpful tool ~ 
I find the function **getpubmedtbl** can't generate the column of date correctly ~

Best 
Huahui

**Comment by yufree** [comment]
Could you share a reproducible example? I just tried the example and the date is fine.

**Comment by rusher321** [comment]
Hope it can reproducible in your enviroment ~ 

```
query <- '((((microbiota) OR (metagenome)) AND (antidiabetic)) AND (human)) AND (("2012/01/01"[Date - Completion] : "3000"[Date - Completion]))'

tmdf <- getpubmed(query) %>%
        getpubmedtbl()

head(tmdf)
```
![image](https://user-images.githubusercontent.com/16647823/103183369-f122bf00-48ec-11eb-8260-e2679b227ad8.png)

```
sessionInfo()
```

R version 3.6.1 (2019-07-05)
Platform: x86_64-w64-mingw32/x64 (64-bit)
Running under: Windows >= 8 x64 (build 9200)

Matrix products: default

locale:
[1] LC_COLLATE=Chinese (Simplified)_China.936 
[2] LC_CTYPE=Chinese (Simplified)_China.936   
[3] LC_MONETARY=Chinese (Simplified)_China.936
[4] LC_NUMERIC=C                              
[5] LC_TIME=Chinese (Simplified)_China.936

**Comment by yufree** [comment]
I still can't reproduce this issue and I can generate the right date.  However, my R version is 4.0.3 and I am not sure if the R version matters.

**Comment by rusher321** [comment]
Thanks , I will update my R and try it again ~

**Comment by yufree** [comment]
Solved through #2
